sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/SdkMeterTest.java

```diff
@@ -20,7 +20,11 @@
 import io.opentelemetry.sdk.testing.exporter.InMemoryMetricReader;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junitpioneer.jupiter.SetSystemProperty;
 
+@SetSystemProperty(
+    key = "org.slf4j.simpleLogger.log.io.opentelemetry.sdk.metrics.internal.state.MeterSharedState",
+    value = "off")
```

Comment:
would it be better to change this to ERROR, rather than disabling altogether?

Reply:
If we were to log an error there would be cases where we have a unit test for it, and we'd probably want off then in that case. Just always using off seems simpler and work well in enough cases to just stick with it

Reply:
I think that makes sense. I'm slightly worried about something creeping in and spamming logs inadvertently, but lets go with it for now.